lib/everypolitician/popolo/organization.rb

```diff
@@ -1,9 +1,49 @@
 module Everypolitician
   module Popolo
     class Organization < Entity
+      def initializer(document)
```

Comment:
what is this method for? Is this meant to be `initialize`? If so, you shouldn't need this, as it will inherit it from `Entity`